apps/judicial-system/backend/src/app/formatters/rulingPdf.ts

```diff
@@ -647,37 +654,38 @@ function constructInvestigationRulingPdf(
     })
   }
 
-  doc
-    .text(' ')
-    .text(
-      `${formatAppeal(existingCase.prosecutorAppealDecision, 'Sækjandi')} ${
-        existingCase.prosecutorAppealDecision === CaseAppealDecision.APPEAL
-          ? existingCase.prosecutorAppealAnnouncement ?? ''
-          : ''
-      }`,
-      {
-        align: 'justify',
-        paragraphGap: 1,
-      },
-    )
+  let prosecutorAppeal = formatAppeal(
+    existingCase.prosecutorAppealDecision,
+    'Sækjandi',
+  )
 
-  // Only show accused appeal decision if applicable
-  if (
-    existingCase.accusedAppealDecision !== CaseAppealDecision.NOT_APPLICABLE
-  ) {
-    doc
-      .text(' ')
-      .text(
-        `${formatAppeal(existingCase.accusedAppealDecision, 'Varnaraðili')} ${
-          existingCase.accusedAppealDecision === CaseAppealDecision.APPEAL
```

Comment:
Import CaseAppealDecision is unused now. Might want to remove that :)

Reply:
Will fix.